packages/website/components/enterpriseTierRequestModal/enterpriseTierRequestModal.js

```diff
@@ -0,0 +1,76 @@
+import { useEffect, useLayoutEffect } from 'react';
+import kwesforms from 'kwesforms';
+
+import Modal from 'modules/zero/components/modal/modal';
+import CloseIcon from 'assets/icons/close';
+import Button from 'components/button/button.js';
+import GradientBackground from 'components/gradientbackground/gradientbackground';
+
+const UserRequestModal = ({ isOpen, onClose }) => {
+  useEffect(() => {
+    kwesforms.init();
```

Comment:
For my own edification, what does this do? i.e. what would be bad if we didn't call it? Does it set up form validation js or something?

Reply:
(if I'm confused, someone else might be. Could be good to put a one-line comment for others that are new to kwesforms)

Reply:
Yeah, I was just copying from the older implementation.  I have no idea.